ios-target/Model/User.swift

```diff
@@ -57,4 +57,19 @@ class User: Codable {
     firstName = try container.decode(String.self, forKey: .firstName)
     lastName = try container.decode(String.self, forKey: .lastName)
   }
+  
+  static func buildParams(user: User) -> [String: Any] {
```

Comment:
neat 👌

Reply:
indeed, but why making it static though? 
do `func buildParams() -> [String: Any] ` and call it on your user object